samples/compute/README.md

```diff
@@ -0,0 +1,204 @@
+instance. These properties are specified in the request body; for details about each property see 
+the [API reference for `instances.insert`](https://cloud.google.com/compute/docs/reference/latest/instances/insert).
+
+At a minimum, your request must provide values for the following properties when you create a new 
+instance:
+
+* Instance name
+* Root persistent disk
+* Machine type
+* Zone
+* Network Interfaces
+
+This sample starts an instance with the following properties in a zone of your choice:
+
+* Machine type: e2-standard-2
+* Root persistent disk: a new persistent disk based on the latest Debian 8 image
+* The Compute Engine default service account with the following scopes:
+  * https://www.googleapis.com/auth/devstorage.read_write, so the instance can read and write files in Cloud Storage
+  * https://www.googleapis.com/auth/logging.write, so the instances logs can upload to Cloud Logging
+* Metadata to specify commands that the instance should execute upon startup
+
+You can see an example of instance creation in the `create_instance` method in [create_instance.py](create_instance.py) file.
+
+### Root persistent disks
+
+All instances must boot from a [root persistent disk](https://cloud.google.com/compute/docs/disks/create-root-persistent-disks).
+The root persistent disk contains all of the necessary files required for starting an instance. 
+When you create a root persistent disk you must select a public image or a custom image to apply to 
+the disk. In the example above, you created a new root persistent disk based on Debian 8 at the same 
+time as the instance. However, it is also possible to create a disk beforehand and attach it to the 
+instance.
+
+To create an instance using your own custom OS image, you need to provide a different URL than 
+the one included in the example. For more information about starting an instance with your own 
+images, see [Creating an instance from a custom image](https://cloud.google.com/compute/docs/instances/create-start-instance#creating_an_instance_from_a_custom_image).
+
+
+
+### Instance metadata
+
+When you create your instance, you might want to include instance metadata such as a [startup script](https://cloud.google.com/compute/docs/startupscript), 
+configuration variables, and SSH keys. In the example above, you used the `metadata` field in your 
+request body to specify a startup script for the instance and some configuration variables as 
+key/values pairs. The startup script, listed below, shows how to read these variables and use them 
+to apply text to an image and upload it to [Cloud Storage](https://cloud.google.com/storage).
+
+The contents of the startup script used in this example can be found in [startup-script.sh](startup-script.sh).
+
+## Deleting an Instance
+
+To delete an instance, you need to call the `instances().delete` method and provide the name, 
+zone, and project ID of the instance to delete. Because you set the `autoDelete` parameter for the 
```

Comment:
>Because you set the `autoDelete` parameter for the 

The user hasn't run the sample yet. That's in the next section `Running the sample`, or perhaps there is a section missing for `Creating an Instance`?

Reply:
I rephrased that to be more universal and not mention any user action.